tests/components/device_tracker/test_entities.py

```diff
@@ -14,19 +14,27 @@
     SOURCE_TYPE_ROUTER,
 )
 from homeassistant.const import ATTR_BATTERY_LEVEL, STATE_HOME, STATE_NOT_HOME
+from homeassistant.helpers import device_registry as dr
 
 from tests.common import MockConfigEntry
 
 
 async def test_scanner_entity_device_tracker(hass, enable_custom_integrations):
     """Test ScannerEntity based device tracker."""
+    # Make device tied to other integration so device tracker entities get enabled
+    dr.async_get(hass).async_get_or_create(
+        name="Device from other integration",
+        config_entry_id=MockConfigEntry().entry_id,
+        connections={(dr.CONNECTION_NETWORK_MAC, "ad:de:ef:be:ed:fe:")},
```

Comment:
```suggestion
        connections={(dr.CONNECTION_NETWORK_MAC, "ad:de:ef:be:ed:fe")},
```

Reply:
I had done that but it's actually bad in the test device tracker integration. I will update it there and see what else breaks.